### ISSUE TITLE
ui-toggle="uiSidebarLeft" not working

### ISSUE THREAD

**Issue by dtytomi** [mcasimir/mobile-angular-ui#333]
Am using mobile-angular-ui with MEANJS boiler-plate and all the design are working except for the app, app-body, and app-content css class that are not working with the boiler-plate but have figured it out. But for the ui-toggle="uiSidebarLeft"  to show the side bar once I resize to mobile screen it doesn't show the sidebar.

**Comment by rubenjimenez** [comment]
Hi,

I'm suffering the same issue. I have not researched a lot yet, but I guess it will be a pain in the ass.

Have you got any shortcut or fix?

**Comment by dtytomi** [comment]
Am sorry no solution still. I did my best, but I couldn't solve it. So, I had to write my own directive.

**Comment by rubenjimenez** [comment]
Hi,

I think I've found the main problem.

Mobile Angular UI uses $rootElement for adding the proper classes to the <body> in order to get the desired effect with sidebars (opening, closing, etc). When ngApp is defined in <body>, everything seems to work like a charm, but when you use angular.bootstrap() (AngularJS manual initialization, used in MEANJS), the $rootElement points to the whole document instead of the <body>, so $rootElement.addClass(whatever) just doesn't work.

I have not studied all the code and I don't know the consequences of using just the element <body> instead of $rootElement, but if there're not any consequences, it'd be desirable.

To sum up, with the current state of the mobile-angular-ui code the fix is to use ngApp in the <body> like they do in their demos or examples.

**Comment by dtytomi** [comment]
Hello @rubenjimenez  how is it going? Well have also made some research based on what you suggested as the likelihood of the cause of why Mobile Angular UI might not be working as it should on MEANjs. So there was this tutorial I saw I believe it might be of help as a way forward. The link is:
http://www.simplygoodcode.com/2014/04/angularjs-getting-around-ngapp-limitations-with-ngmodule/

**Comment by rubenjimenez** [comment]
Hi @dtytomi, thanks for the tutorial. I realized of that a little while after I published my answer here.

I found that indeed ngApp was not the best way to fix our problem. I kept searching and I found this issue  #222, where they explain it's better to use angular.bootstrap(), just like your tutorial says.

So, replacing the current angular.bootstrap of MEANJS from
```
angular.bootstrap(document, ['mean']);
```
to
```
angular..bootstrap(document.body, ['mean']);
```
did the trick.

I hope this was useful for you =)

**Comment by dtytomi** [comment]
angular.bootstrap(document.body, ['mean']);
